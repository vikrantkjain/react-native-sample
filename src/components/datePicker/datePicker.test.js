import React from 'react';
import DatePicker from './index';
import {render} from '@testing-library/react-native';

const onDateChange = jest.fn();
// jest.mock('Platform', () => {
//   const Platform = require.requireActual('Platform');
//   Platform.OS = 'ios';
//   return Platform;
// });
describe('Test cases for date picker component', () => {
  const tree = render(
    <DatePicker date={new Date()} mode="date" onDateChange={onDateChange} />,
  );
  it('render successfully', () => {
    expect(tree).toBeDefined();
  });
  it('render correctly', () => {
    const treeAsJSON = tree.toJSON();
    expect(treeAsJSON).toMatchSnapshot();
  });
});
