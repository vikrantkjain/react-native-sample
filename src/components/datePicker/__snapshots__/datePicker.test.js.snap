// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test cases for date picker component render correctly 1`] = `null`;
